debian/rules

```diff
@@ -108,9 +108,10 @@ ifeq (,$(filter nocheck,$(DEB_BUILD_OPTIONS)))
 	# check that only the main trusted account-key is included
```

Comment:
Not strictly related to this PR but I think the lines before this check are slightly confusing. I think we should check there for "testkeys"  instead of "nocheck". I.e. write it like:
```
index 3dfad9a..494cab6 100755
--- a/debian/rules
+++ b/debian/rules
@@ -102,13 +102,15 @@ override_dh_auto_build:
 
 override_dh_auto_test:
        dh_auto_test -- $(GCCGOFLAGS)
-ifeq (,$(filter nocheck,$(DEB_BUILD_OPTIONS)))
+ifeq (,$(filter testkeys,$(DEB_BUILD_OPTIONS)))
        # check that only the main trusted account-key is included
        [ $$(strings _build/bin/snapd|grep -c -E "public-key-sha3-384: [a-zA-Z0-9_-]{64}") -eq 1 ]
        strings _build/bin/snapd|grep -c "^public-key-sha3-384: -CvQKAwRQ5h3Ffn10FILJoEZUXOv6km9FwA80-Rcj-f-6jadQ89VRswHNiEB9Lxk$$"
 endif
+ifeq (,$(filter nocheck,$(DEB_BUILD_OPTIONS)))
        # run the snap-confine tests
        $(MAKE) -C cmd check
+endif
 
 override_dh_systemd_enable:
        # enable auto-import
```
WDYT?